WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
Calm down first of all. Secondly you can look at what the code already includes, namely the Christian religion group, which is why I'm sceptic to your change.